charts/argo-controller/values.schema.json

```diff
@@ -79,6 +79,16 @@
         }
       },
       "required": ["podGarbageCollection", "workflowTTL"]
+    },
+    "metrics": {
+      "type": "object",
+      "properties": {
+        "enabled": { "type": "boolean" },
+        "path": { "type": "string" },
+        "port": { "type": "integer" },
+        "label": { "type": "string" }
```

Comment:
This `label` isn't needed any more, it should be removed from the properties and the required list.